packages/atlas/src/config/categories.tsx

```diff
@@ -1,113 +1,146 @@
+    coverImgUrl: 'https://eu-central-1.linodeobjects.com/atlas-assets/category-images/comedy.webp',
+    videoCategories: ['59-2'],
   },
-  '2': {
-    id: '2',
-    icon: <SvgCategoriesAutosAndVehicles />,
+  {
+    id: '10',
+    name: 'Eleventh category',
+    iconUrl: 'https://dl.dropboxusercontent.com/s/q6jwehpxwido4q2/categories-autos-and-vehicles.svg?dl=0',
     color: '#6E5FEC',
-    coverImg: 'https://eu-central-1.linodeobjects.com/atlas-assets/category-images/autos-and-vehicles.webp',
+    coverImgUrl: 'https://eu-central-1.linodeobjects.com/atlas-assets/category-images/autos-and-vehicles.webp',
+    videoCategories: ['61-2', '59-2', '63-2'],
   },
-  '12': {
-    id: '12',
-    icon: <SvgCategoriesHowtoAndStyle />,
+  {
+    id: '11',
+    name: 'Twelfth category',
+    iconUrl: 'https://dl.dropboxusercontent.com/s/6no58hs9o9l9n18/categories-howto-and-style.svg?dl=0',
     color: '#E57827',
-    coverImg: 'https://eu-central-1.linodeobjects.com/atlas-assets/category-images/howto-and-style.webp',
+    coverImgUrl: 'https://eu-central-1.linodeobjects.com/atlas-assets/category-images/howto-and-style.webp',
+    videoCategories: ['61-2', '63-2'],
   },
-  '3': {
-    id: '3',
-    icon: <SvgCategoriesMusic />,
+  {
+    id: '12',
+    name: 'Thirteenth category',
+    iconUrl: 'https://dl.dropboxusercontent.com/s/07058sh62h9wsf8/categories-music.svg?dl=0',
     color: '#6EEC3A',
-    coverImg: 'https://eu-central-1.linodeobjects.com/atlas-assets/category-images/music.webp',
+    coverImgUrl: 'https://eu-central-1.linodeobjects.com/atlas-assets/category-images/music.webp',
+    videoCategories: ['59-2', '65-2'],
   },
-  '15': {
-    id: '15',
-    icon: <SvgCategoriesNonprofitsAndActivism />,
+  {
+    id: '13',
+    name: 'Fourteenth category',
+    iconUrl: 'https://dl.dropboxusercontent.com/s/mgs2h2657plqc74/categories-nonprofits-and-activism.svg?dl=0',
     color: '#E141D6',
-    coverImg: 'https://eu-central-1.linodeobjects.com/atlas-assets/category-images/nonprofit-and-activism.webp',
+    coverImgUrl: 'https://eu-central-1.linodeobjects.com/atlas-assets/category-images/nonprofit-and-activism.webp',
+    videoCategories: ['63-2', '65-2', '59-2', '61-2'],
   },
-  '11': {
-    id: '11',
-    icon: <SvgCategoriesNewsAndPolitics />,
+  {
+    id: '14',
+    name: 'Fifteenth category',
+    iconUrl: 'https://dl.dropboxusercontent.com/s/amjwibevfqav8n4/categories-news-and-politics.svg?dl=0',
     color: '#48F0B3',
-    coverImg: 'https://eu-central-1.linodeobjects.com/atlas-assets/category-images/news-and-politics.webp',
+    coverImgUrl: 'https://eu-central-1.linodeobjects.com/atlas-assets/category-images/news-and-politics.webp',
+    videoCategories: ['63-2'],
   },
-} as const
+]
+
+export const findDisplayCategory = (id?: string | null) => {
+  const displayCategoriesLookup = createLookup(displayCategories)
+  if (!id) return null
+  return displayCategoriesLookup[id]
+}
```

Comment:
Why do you create lookup inside a function? This isn't very performant - the list of categories is constant so we can create the lookup only once, as top-level const in this file and just import it directly when needed.